sites/kit.svelte.dev/src/routes/docs/[slug].js

```diff
@@ -0,0 +1,28 @@
+import { mode, prerendering } from '$app/env';
+import { read } from '$lib/docs/server';
+
+/** @type {import('@sveltejs/kit').RequestHandler} */
+export async function get({ params }) {
+	// TODO https://github.com/sveltejs/kit/issues/4093
+	if (!prerendering && mode === 'production') {
```

Comment:
Is there a reason this is checking for `mode === 'production'` rather than `!dev`?

Reply:
yes, me forgetting that such a thing existed